iconservice/icon_service_engine.py

```diff
@@ -1336,6 +1343,13 @@ def _handle_get_iiss_info(self, context: 'IconScoreContext', _params: dict) -> d
 
         return response
 
+    def _handle_get_service_config(self,
+                                   context: 'IconScoreContext',
+                                   _params: dict) -> dict:
+        response = dict()
+        response['config'] = self._conf
```

Comment:
1. Are default values included to this response?
2. How about reducing one depth? "config" depth looks redundant.